odoo/addons/base/tests/test_translate.py

```diff
@@ -247,6 +247,7 @@ class TestTranslation(TransactionCase):
 
     def setUp(self):
         super(TestTranslation, self).setUp()
+        self.env['res.lang'].search([('code', '!=', 'en_US'), ('active', '=', True)]).active = False
```

Comment:
Why ?

Reply:
@tde-banana-odoo  Because the tests assume that only one lang is activated (en_US), so a lot of tests fail when I activate some other languages by default